src/razor/src/semantic/provideSemanticTokensResponse.ts

```diff
@@ -5,5 +5,5 @@
 
 export class ProvideSemanticTokensResponse {
     // tslint:disable-next-line: variable-name
-    constructor(public Tokens: Array<Array<number>>, public HostDocumentSyncVersion: number) {}
+    constructor(public Tokens: Uint32Array, public HostDocumentSyncVersion: number) {}
```

Comment:
This is a `Uint32Array` since that's what `vscode.SemanticTokens.data` uses as their type

Reply:
Isn't this a mismatch to the Razor request type though? @maryamariyan made it return an array of arrays when doing the precise tokens work. I'm surprised this doesn't fail to deserialize on the Razor side

Reply:
It looks like the feature flag is off in VS Code? I'm seeing `UsePreciseSemanticTokenRanges` return false

Reply:
Is the flag meant to be on in VS Code? @maryamariyan

Reply:
Looks like the type changed back to a single dimension array in the latest updates on the Razor side. Maybe we just never got around to updating this side of things, since it didn't do anything.

Reply:
Okay, latest Razor bump was 5 days ago in https://github.com/dotnet/vscode-csharp/pull/6449 and the new Razor server changes went in 7 days ago, so this change to a single dimensional array makes sense.